### PR TITLE
Awesome Books with ES6

### PR DESCRIPTION
**Pull request by HansZizold**
An entry point (index.js) for the project was created.
An index.css file was created for styles.
Index.js and index.css were linked in index.html.
ES6 syntax was used (let, const and arrow functions).
The original js file was divided into modules, these modules were stored in the 'modules' folder.
All functionalities are maintained without errors.
Luxon library was used as a module to display the date and time below the navigation bar.

Corrections: default option used when exporting the modules and linters corrected.

### CODE REVIEW

File: modules/addNewBook.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/* eslint-disable import/prefer-default-export */
```
Comment (rloterh) [inline modules/addNewBook.js]: - [x] You have disabled Eslint checks on your code. Whiles your project looks great **please enable all linter checks** on your code to ensure that your code is robust code and clean.

File: modules/addNewBook.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+const addmybooks = document.querySelector('.addmybooks');
+const contactInfo = document.querySelector('.contact-info');
+const bookshelf = document.querySelector('.bookshelf');
+
+// showing the add my book section and hiding the rest
+// export function addNewBook() {
```
Comment (rloterh) [inline modules/addNewBook.js]: - [ ] OPTIONAL
There is commented code block found in your code on the repository. Kindly endeavor to remove **all** commented code from your code before pushing it to the repository so as to keep your code well organized.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Hans, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Highlights 👍  
- Descriptive Pull Request
- ES6 syntax correctly used
- Module import correctly implemented
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comments marked **[OPTIONAL]** under the review._

You can also consider:
N/A

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/addNewBook.js (hunk 1 above)
Comment (HansZizold) [reply modules/addNewBook.js]: Very sorry for that. I enabled linters.

File: modules/addNewBook.js (hunk 2 above)
Comment (HansZizold) [reply modules/addNewBook.js]: Thank you for the observation. I deleted the line.

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @HansZizold, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- You have done a great job working on the requested changes ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**